src/main/java/nl/ordina/jobcrawler/service/LocationService.java

```diff
@@ -0,0 +1,121 @@
+                }
+            }
+        }
+        return Optional.empty();
+    }
+
+    public static Location getCoordinates(String location) throws IOException, JSONException {
+        final String apiKey = "Xd5hXSuQvqUJJbJh3iacOXZAcskvP7gI";
+        String location2 = location.concat(", Nederland");
+        location2 = location2.replace(" ","%20");
+        final String url = "http://open.mapquestapi.com/nominatim/v1/search.php?key=" + apiKey + "&format=json&q=" + location2 + "&addressdetails=1&limit=1";
+
+        URL obj = new URL(url);
+        HttpURLConnection connection = (HttpURLConnection) obj.openConnection();
+        // optional default is GET
+        connection.setRequestMethod("GET");
+        //add request header
+        connection.setRequestProperty("User-Agent", "Mozilla/5.0");
+        if (connection.getResponseCode() == 200) {
+/*            System.out.println("\nSending 'GET' request to URL : " + url);
+            System.out.println("Response Code : " + connection.getResponseCode());*/
+            BufferedReader in = new BufferedReader(
+                    new InputStreamReader(connection.getInputStream()));
+            String inputLine;
+            String response = in.readLine();
+            response = response.substring(1, response.length() - 1);
+            System.out.println(response);
+            in.close();
+            //Read JSON response and return
+            JSONObject jsonResponse = new JSONObject(response);
+
+            return new Location(location, jsonResponse.getDouble("lon"), jsonResponse.getDouble("lat"));
+
+        }
+        return new Location(location, 0, 0);
+    }
+
+    public static double getDistanceFromHome(Location homeLocation, Location vacancyLocation) {
+        double lon1 = homeLocation.getLon();
+        double lat1 = homeLocation.getLat();
+        double lon2 = vacancyLocation.getLon();
+        double lat2 = vacancyLocation.getLat();
+        System.out.println(Math.sqrt(Math.pow((lon2 - lon1), 2) + Math.pow((lat2 - lat1), 2)) * 100);
+        // The math module contains a function
+        // named toRadians which converts from
+        // degrees to radians.
+        lon1 = Math.toRadians(lon1);
+        lon2 = Math.toRadians(lon2);
+        lat1 = Math.toRadians(lat1);
+        lat2 = Math.toRadians(lat2);
+        // Haversine formula
+        double dlon = lon2 - lon1;
+        double dlat = lat2 - lat1;
+        double a = Math.pow(Math.sin(dlat / 2), 2)
+                + Math.cos(lat1) * Math.cos(lat2)
+                * Math.pow(Math.sin(dlon / 2), 2);
+        double c = 2 * Math.asin(Math.sqrt(a));
+        // Radius of earth in kilometers. Use 3956 for miles
+        double r = 6371;
+        // calculate the result
+        return (c * r);
+    }
+
+    public List<Location> findAll() {
+        return locationRepository.findAll();
+    }
+
+    public Location update(UUID uuid, Location location) {
+        return null;
+    }
```

Comment:
Method was added because of other services extending the crudservice (the one that was recently removed)? As it's not mandatory to add this method you can remove it.